js/utils.js

```diff
@@ -5,6 +5,8 @@ export const usbPortFilters = [
     { usbVendorId: 0x303a, usbProductId: 0x1002 }, /* Espressif esp-usb-bridge firmware */
     { usbVendorId: 0x303a, usbProductId: 0x0002 }, /* ESP32-S2 USB_CDC */
     { usbVendorId: 0x303a, usbProductId: 0x0009 }, /* ESP32-S3 USB_CDC */
+    { usbVendorId: 0x1a86, usbProductId: 0x55d4 },
+    { usbVendorId: 0x1a86, usbProductId: 0x7523 },
```

Comment:
Could you please add trailing comments, similar to the lines above?

Reply:
@igrr, I have added trailing comments as per suggestion. PTAL 😄